### PR TITLE
Fix flaky test test_multiple_disks/test.py::test_rename

### PR DESCRIPTION
**Pull request by kssenii**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Review by alexey-milovidov — APPROVED** [review]

**Comment by qoega** [comment]
>           assert len(disks) > 1
E           NameError: name 'disks' is not defined

File: tests/integration/test_multiple_disks/test.py [hunk 1]
```diff
@@ -1635,7 +1636,13 @@ def test_rename(start_cluster):
...
 
-        disks = get_used_disks_for_table(node1, "renaming_table")
+        # data is moved in the background, so check with retries
+        num_try = 0
+        while get_used_disks_for_table(node1, "renaming_table") == 1:
+            time.sleep(1)
+            num_try += 1
+            if num_try == 20:
+                break
         assert len(disks) > 1
```
Comment (qoega) [inline tests/integration/test_multiple_disks/test.py:1646]:
```suggestion
        assert len(get_used_disks_for_table(node1, "renaming_table")) > 1
```

**Comment by qoega** [comment]
Also failed `test_multiple_disks/test.py::test_background_move[mt]`

```
>           assert used_disks[0] == "external"
E           AssertionError: assert 'jbod1' == 'external'
E             - external
E             + jbod1
```
https://s3.amazonaws.com/clickhouse-test-reports/0/cfba9b19eb0941f4e40d216d55ef2a68f6d2cf5e/integration_tests__release__[2/2]/integration_run_parallel5_0.log

**Comment by kssenii** [comment]
> Also failed test_multiple_disks/test.py::test_background_move[mt]

the reason is most likely the same as in test_rename which I fixed in this PR, will fix..

**Pull request merged**